docs/source/bundle.rst

```diff
@@ -32,6 +32,7 @@ Model Bundle
 ---------------
 .. autoclass:: ConfigParser
     :members:
+    :special-members:
```

Comment:
May I know why add this line?

Thanks.

Reply:
special member functions are not in the autogenerated doc by default https://docs.monai.io/en/stable/bundle.html#monai.bundle.ConfigParser